src/usr/local/buildpack/tools/git.sh

```diff
@@ -14,4 +14,4 @@ curl --retry 3 -sSL \
 
 apt_install git
 
-git --version
+SKIP_VERSION || git --version
```

Comment:
```suggestion
$SKIP_VERSION || git --version
```
isn't it a environment variable?